.storybook/manager.ts

```diff
@@ -0,0 +1,23 @@
+import { addons } from '@storybook/manager-api';
+
+addons.setConfig({
+  navSize: 300,
+  rightPanelWidth: 300,
+  panelPosition: 'right',
```

Comment:
The other options are set as default values.
However, I set `panelPosition` from `bottom` to `right`